MarkdownView/markdownview/build.gradle

```diff
@@ -8,27 +8,19 @@ ext {
 
 android {
     compileSdkVersion 23
-    buildToolsVersion "23.0.1"
+    buildToolsVersion "23.0.2"
 
     defaultConfig {
         minSdkVersion 14
-        targetSdkVersion 23
```

Comment:
Library projects should not set a targetSdkVersion, since that can create dependency conflicts for consumers who's app target version is different.